components/common/src/command/package/install.rs

```diff
@@ -176,7 +181,19 @@ impl<'a> InstallTask<'a> {
             debug!("Found {} in artifact cache, skipping remote download",
                    &ident);
         } else {
-            try!(self.fetch_artifact(ui, &ident, src_path));
+            if retry(RETRIES,
+                     RETRY_WAIT,
+                     || self.fetch_artifact(ui, &ident, src_path),
+                     |res| res.is_ok())
+                .is_err() {
+                return Err(Error::from(depot_client::Error::DownloadFailed(format!("We tried {} \
```

Comment:
Just an opinion - maybe pulling the `format!` out of `Error::from` and assigning it to a variable could make it a bit more readable with fewer shifted line breaks\continuations.